src/lib/components/NewLayeredMap/layers/L.imageWebGLOverlay.js

```diff
@@ -94,6 +94,17 @@ L.ImageWebGLOverlay = L.Layer.extend({
 		return this;
     },
     
+    reUseColorscale: function (newOptions) {
+        const newColorScaleOptions = newOptions.colorScale;
+        const updatedColorScale = this.options.colorScale;
+
+        if(this.options.colorScale) {
```

Comment:
Can easily be replaced with `Object.assign({}, ...)`.

In addition, the colorScale-property is not necessarily an object. It might be an object and might be null.